acados/ocp_nlp/ocp_nlp_common.h

```diff
@@ -117,6 +118,7 @@ typedef struct ocp_nlp_config
 
     // config structs of submodules
     ocp_qp_xcond_solver_config *qp_solver; // TODO rename xcond_solver
+    ocp_qp_xcond_solver_config *relaxed_qp_solver;
```

Comment:
why is this in `nlp_common`? Don't we need the relaxed QP solver only with `SQP_WITH_FEASIBLE_QP`?